src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoHandler.php

```diff
@@ -171,16 +172,24 @@ public static function getOrderSystemStatus(): array
         $orderSystemStatus['last_synchronized_date'] = TaskModel::getLatestSuccessfulSynchronizedDateForType(TaskHandler::ORDERS_EXPORT);
 
         $orderSystemStatus['ids_with_failed_tasks'] = TaskModel::getFailedOrderIds();
-        $unsynchronizedOrders = wc_get_orders([
+
+        $orderStatuses = wc_get_order_statuses();
+        unset($orderStatuses['wc-cancelled']);
+        $orderQuery = new \WC_Order_Query([
             'meta_key' => OrderHandler::TO_BE_SYNCHRONIZED_META_KEY,
             'meta_value' => 'yes',
             'meta_compare' => '=',
             'orderby' => 'date_created',
             'order' => 'ASC',
         ]);
-
+        $orderQuery->set('status', array_keys($orderStatuses));
+        $unsynchronizedOrders = $orderQuery->get_orders();
         $unsynchronizedOrderIds = [];
 
+        $unsynchronizedOrders = array_filter($unsynchronizedOrders, static function (WC_Order $order) {
+            return OrderExport::STATUS_CANCELLED !== $order->get_status();
```

Comment:
why do we need to filter it is we do not query for it?

Reply:
We have the [query](https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/pull/114/files#diff-43261a512e63739d6d9eee3ed4c80b0acc5c6a1284f1926b7c1ce8f526f897a0R185) for it but it does not work so I had to add this. I tried looking at docs and forums and that's how they filter by status but for us it doesn't work. I don't know why.